### PR TITLE
Added documentation for all the methods and changed hook url for flak…

### PR DESCRIPTION
**Pull request by bharath-newpage**
…e8 in pre-commit yml

### CODE REVIEW

File: src/enhanced_reports/common_utils.py [hunk 1]
```diff
@@ -9,16 +9,32 @@
 
 
 def get_resized_resolution(width, height, resize_factor) -> Tuple[int, int]:
```
Comment (davidnewpage) [inline src/enhanced_reports/common_utils.py:11]: still missing type hint here?

File: src/enhanced_reports/common_utils.py [hunk 2]
```diff
@@ -9,16 +9,32 @@
...
+    @param height: Height to resize.
+    @param resize_factor: Refactor size of an image in the form of % Ex: 0.1.
+    @return: New width and height in Tuple format
+    """
     new_width = int(width * resize_factor)
     new_height = int(height * resize_factor)
     return new_width, new_height
 
 
 def mkdir(dir_name):
```
Comment (davidnewpage) [inline src/enhanced_reports/common_utils.py:24]: same here

File: src/enhanced_reports/common_utils.py (hunk 1 above)
Comment (bharath-newpage) [reply src/enhanced_reports/common_utils.py]: Added method params

File: src/enhanced_reports/common_utils.py (hunk 2 above)
Comment (bharath-newpage) [reply src/enhanced_reports/common_utils.py]: Added

File: src/enhanced_reports/core.py [hunk 3]
```diff
@@ -41,14 +41,16 @@
 
 # region Registering and retrieving config parameters
 def pytest_addoption(parser: argparsing.Parser):
-    logger.debug("Entered " + inspect.currentframe().f_code.co_name)
+    logger.debug("Entered Adoption" + inspect.currentframe().f_code.co_name)
```
Comment (swaraj-np) [inline src/enhanced_reports/core.py:44]: it is "add"option

Comment (swaraj-np) [reply src/enhanced_reports/core.py:44]: but, more importantly, if you are going to do it like this, then the ` + inspect.currentframe().f_code.co_name` is not needed. it does the same thing. this particular line for example would log the string "Entered Adoption pytest_addoption"

Comment (swaraj-np) [reply src/enhanced_reports/core.py:44]: you can either hard code the method name or use the inspect module to get the method name. not both

Comment (davidnewpage) [reply src/enhanced_reports/core.py:44]: resolved

**Review by swaraj-np — APPROVED** [review]

**Pull request merged**